pkg/grpc/grpc.go

```diff
@@ -77,7 +75,6 @@ func (g *Manager) GetGRPCConnection(address, id string, skipTLS, recreateIfExist
 	}
 
 	opts := []grpc.DialOption{
-		grpc.WithBlock(),
```

Comment:
We need this, otherwise, there should be unnecessary retry.

Reply:
reverted